README.md

```diff
@@ -158,6 +158,12 @@ The above command pulled the existing data out of Route53 and placed the results
 | [TinyDNSSource](/octodns/source/tinydns.py) | A, CNAME, MX, NS, PTR | No | read-only |
 | [YamlProvider](/octodns/provider/yaml.py) | All | Yes | config |
 
+#### Notes
+
+* ALIAS support varies a lot fromm provider to provider care should be taken to verify that your needs are met in detail.
+   * Dyn's UI doesn't allow editing or view of TTL, but the API accepts and stores the value provided, this value does not appear to be used when served
+   * Dnsimple's API throws errors when TTL is modified, but it can be edited in the UI and seems to be used when served, there's also a secondary TXT record created alongside the ALIAS that octoDNS ignores
```

Comment:
Not sure. It might be possible, but the hiccup would be that it may not be clear that it's an `ALIAS` record at the point the error is thrown. Would have to dig into it more.